web/modules/custom/dbcdk_community/src/Form/QuarantineEditForm.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\dbcdk_community\Form\QuarantineEditForm.
```

Comment:
The two forms have been merged to one but I've not instantiated an empty `Quarantine` in the constructor since it seemed more clean to check the ::buildForm if "Quarantine" was empty instead of checking for empty values in the empty `Quarantine` - I do it in [::submitForm() instead](https://github.com/DBCDK/biblo-admin/pull/38/files#diff-f2a84a73f7355c1bbbdd2530354f88e1R165).